Tensile/Tests/extended/local_split_u/sgemm_lsu_mfma.yaml

```diff
@@ -48,7 +48,8 @@ BenchmarkProblems:
         - GlobalSplitUWorkGroupMappingRoundRobin: [False]
         - GlobalSplitUSummationAssignmentRoundRobin:  [True]
         - 1LDSBuffer: [0]
-        - LdsBlockSizePerPad: [128]
+        - LdsBlockSizePerPadA: [-1]
+        - LdsBlockSizePerPadB: [-1]
```

Comment:
Do we need a test for setting specific values like this test used to do?

Reply:
We have some tests with LBSPP=128.